### ISSUE TITLE
nus-wide/resnet101.pth loads fail

### ISSUE THREAD

**Issue by N0Carrot** [deeplearning-wisc/multi-label-ood#6]
When I load the nus-wide/resnet101.pth file, the code reports an error that Error(s) in loading state_dict for ResNet: Missing key(s) in state_dict.
